modules/op-installing-tekton-hub-with-login-and-rating.adoc

```diff
@@ -0,0 +1,110 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="installing-tekton-hub-with-login-and-rating.adoc_{context}"]
+= Installating {tekton-hub} with login and rating
+
+[role="_abstract"]
+You can install {tekton-hub} on your cluster with custom configuration that supports login with authorization and ratings for {tekton-hub} artifacts.
+
+[discrete]
+.Prerequisites
+* Ensure that the {pipelines-title} Operator is installed in the default `openshift-pipelines` namespace on the cluster.
+
+[discrete]
+.Procedure 
+
+. Create a fork of the link:https://github.com/tektoncd/hub[{tekon-hub}] repository.
```

Comment:
In the preview,
![Screenshot from 2022-11-02 20-32-11](https://user-images.githubusercontent.com/13780748/199524903-bcdb35bd-4863-4f29-8697-77211f10fa70.png)
 it is shown like this. Is this fine?